### DISCUSSION TITLE
No response on second round from Campus Expert Application

### DISCUSSION THREAD

**Discussion by Vikash-8090-Yadav** [community/community#52986] [GitHub Education] [answered]
Hello GitHub Education team,

I applied for the GitHub Campus Expert program during the February 2023 batch and was selected for the next round, which involved submitting a video resume. It has been two weeks since the submission closed, but I have not received any updates on my application status yet. I am very excited about this opportunity and couldn't resist reaching out to inquire about when I can expect to hear back.

Thank you for your time and consideration. I apologize for any errors in my previous message.

Best regards.

**Comment by Abdelaziz-LAISSAOUI** [comment]
same goes for me I am still waiting for the response

**Reply by Vikash-8090-Yadav** [reply]
> same goes for me I am still waiting for the response

Yes, bro !!  It's 3 more days now but I didn't get any response from the GitHub team side!
Let's hope for the best .

**Reply by fierylion** [reply]
Same here brother!. I am eagerly waiting to be part of the program

**Reply by Vikash-8090-Yadav** [reply]
I would recommend u all to please vote for this discussion by tapping up arrow icon. So that more and more can see this and join our discussion . @fierylion @Abdelaziz-LAISSAOUI 


![Screenshot from 2023-04-17 19-35-46](https://user-images.githubusercontent.com/85225156/232509094-79cd3d49-7d48-4e3e-ba1e-de6dfd46a506.png)

**Reply by Abdelaziz-LAISSAOUI** [reply]
done ✅

**Reply by fierylion** [reply]
Done💯

**Comment by Memomer** [comment]
Done

**Comment by Vikash-8090-Yadav** [comment]
@fierylion @Memomer @Abdelaziz-LAISSAOUI @zackfern  anyone got any updates ?

**Reply by Abdelaziz-LAISSAOUI** [reply]
I have just checked my email and still no response

**Reply by Vikash-8090-Yadav** [reply]
> I have just checked my email and still no response 

Same 🥲

**Reply by fierylion** [reply]
Same, no response

**Comment by KenjiPcx** [comment]
Hi, has anyone heard back yet?

**Reply by fierylion** [reply]
Not yet brother

**Reply by Abdelaziz-LAISSAOUI** [reply]
not yet

**Comment by Vikash-8090-Yadav** [comment, marked as answer]
Hey all , The result is out now . Check ur mail ✌️

**Reply by fierylion** [reply]
I got the reply, Thanks everyone Hope to meet all of you there😀😍

**Reply by Abdelaziz-LAISSAOUI** [reply]
see you there brothers 😀
